src/libraries/System.Net.Security/src/System.Net.Security.csproj

```diff
@@ -18,6 +18,7 @@
     <DefineConstants>$(DefineConstants);SYSNETSECURITY_NO_OPENSSL</DefineConstants>
   </PropertyGroup>
   <ItemGroup Condition="'$(TargetsAnyOS)' != 'true'">
+    <Compile Include="System\Net\Security\MD4.cs" />
```

Comment:
This should be only included on OSes that actually need it. 

It may be best to omit this line from this PR, and only add it later together with the actual use.

Reply:
I think we should condition this for android only.

Reply:
If its only for Android, then would belong under the ItemGroup that is conditioned on `UseAndroidCrypto`?

Reply:
updated, thanks